docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/logo/logo.twig

```diff
@@ -5,36 +5,71 @@
  *
  * Variables:
  * - theme: [string] Theme: light, dark.
+ * - type: [string] Type: default, inline, stacked.
+ * - with_secondary_image: [boolean] Has secondary image.
  * - logos: [object] Object with 'mobile' and 'desktop' keys:
  *    - src: [string] URL for the logo.
  *    - alt: [string] Image alt text.
+ * - logo_secondary: [object] Optional object with 'mobile' and 'desktop' keys:
+ *    - src: [string] URL for the logo.
+ *    - alt: [string] Image alt text.
  * - url: [string] Optional URL that wraps the logo.
  * - title: [string] Optional logo title attribute.
  * - attributes: [string] Additional attributes.
  * - modifier_class: [string] Additional classes.
  */
 #}
 
+{% set type = type in ['default', 'inline', 'stacked'] ? type : 'default' %}
+{% set type_class = 'civictheme-logo-type-%s'|format(type|default('default')) %}
 {% set theme_class = 'civictheme-theme-%s'|format(theme|default('light')) %}
-{% set modifier_class = '%s %s'|format(theme_class, modifier_class|default('')) %}
+{% set logo_secondary_class = with_secondary_image ? 'civictheme-logo-secondary' : '' %}
+{% set modifier_class = '%s %s %s %s'|format(theme_class, logo_secondary_class, type_class, modifier_class|default('')) %}
 
 {% if logos is not empty %}
-  {% for type, logo in logos %}
-    {% set visibility_class = type == 'mobile' ? 'show-xxs hide-m' : 'hide-xxs show-m' %}
-    {% if logo.src is not empty %}
+  {% if with_secondary_image and logo_secondary is not empty and type != 'default' %}
+    {% set logos = [logos, logo_secondary] %}
+  {% else %}
+    {% set logos = [logos] %}
+  {% endif %}
+  {% set logo_mobile_components = {} %}
+  {% set logo_desktop_components = {} %}
+  {% for logo in logos %}
+    {% if loop.first %}
+      {% set logo_mobile_components = logo_mobile_components|merge({'logo': logo.mobile}) %}
```

Comment:
@joshua-salsadigital this is going into the wrong direction :))

The Logo component knows only about 3 things:
- Theme (Dark, Light)
- Type: default, inline, stacked
- Object with logos:
`logos.primary.src` and `logos.primary.alt`; `logos.secondary.src` and `logos.secondary.alt`


The storybook itself should be able to load those assets and feed them into this component.